### PR TITLE
Enable setting various DB types on init process

### PR DESCRIPTION
**Pull request by aidan-kwon**
## Proposed changes

Change the followings during `init` genesis block. 
- Set `dbtype` to option value
- Don't generate "lightchaindata" database which is not used in Klaytn

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: cmd/utils/nodecmd/chaincmd.go [hunk 1]
```diff
@@ -141,10 +141,25 @@ func initGenesis(ctx *cli.Context) error {
...
+			TableName:          ctx.GlobalString(utils.DynamoDBTableNameFlag.Name),
+			Region:             ctx.GlobalString(utils.DynamoDBRegionFlag.Name),
+			IsProvisioned:      ctx.GlobalBool(utils.DynamoDBIsProvisionedFlag.Name),
+			ReadCapacityUnits:  ctx.GlobalInt64(utils.DynamoDBReadCapacityFlag.Name),
+			WriteCapacityUnits: ctx.GlobalInt64(utils.DynamoDBWriteCapacityFlag.Name),
+			ReadOnly:           ctx.GlobalBool(utils.DynamoDBReadOnlyFlag.Name),
+		}
+	}
+
+	for _, name := range []string{"chaindata"} { // Remove "lightchaindata" since Klaytn doesn't use it
```
Comment (winnie-byun) [inline cmd/utils/nodecmd/chaincmd.go:159]:
```suggestion
	for _, name := range []string{"chaindata"} { // Removed "lightchaindata" since Klaytn doesn't use it
```

File: storage/database/dynamodb.go [hunk 2]
```diff
@@ -373,7 +373,7 @@ func (dynamo *dynamoDB) Close() {
 	if dynamoOpenedDBNum > 0 {
 		dynamoOpenedDBNum--
 	}
-	if dynamoOpenedDBNum == 0 {
+	if dynamoOpenedDBNum == 0 && dynamoWriteCh != nil {
```
Comment (ehnuje) [inline storage/database/dynamodb.go:376]: Is this related to this PR?

**Review by winnie-byun — APPROVED** [review]

File: storage/database/dynamodb.go (hunk 2 above)
Comment (aidan-kwon) [reply storage/database/dynamodb.go:376]: Little bit. 
`dynamoWriteCh` is shared by all DynamoDB databases. If I didn't remove "lightchaindata" from DB names or another DB name is introduced, `dynamoWriteCh` is forced to be closed more than one time.

**Review by ehnuje — APPROVED** [review]

**Review by jeongkyun-oh — APPROVED** [review]

**Review by yoomee1313 — APPROVED** [review]

**Review by KimKyungup — APPROVED** [review]
